.github/workflows/cmake.yml

```diff
@@ -14,8 +14,8 @@ permissions:
 env:
   # Must be updated manually. Increment 'TD_MARK' to invalidate the cache.
   # Must not contain spaces, minuses, or any special characters.
-  TD_TAG: "v1.8.0"
-  TD_MARK: "m1"
+  TD_TAG: "1fa2a372a88c26369dcac2ce476166531df74a77"
```

Comment:
So are we using td@1fa2a372a88c26369dcac2ce476166531df74a77 now?

Reply:
At the moment of publish pull request the latest version was 1.8.34 and when I building on new version, I've checking tdlib repository for commit message 'Update version to 1.x.yy' and getting commit id for it. 1fa2a372a88c26369dcac2ce476166531df74a77 hash matching the commit https://github.com/tdlib/td/commit/1fa2a372a88c26369dcac2ce476166531df74a77

I've also built and tested with 1.8.35 that released some later after thert PR, no issues found, so  I could update this PR to commit matching 1.8.35.

I agree that is not good to use commit id. Unfortunately tdlib developers does not use tags for minor releases, currently the latest tag still 1.8.0 (see https://github.com/tdlib/td/tags), but tdlib since 1.8.0 has a lot of incompatible changes.

Reply:
It's fine to use commit hashes, I know that the td devs are highly unconventional. I was questioning the fact that it was two *different* commits, which is now fixed. Thank you!

Reply:
Updating it is a good idea. Just dropping it is a bad idea, because the project already is dangerously low on tests.